fractal/client/cmd/_workflow.py

```diff
@@ -115,10 +116,10 @@ async def workflow_edit_task(
 
     with Path(json_file).open("r") as f:
         payload = json.load(f)
-
+    payload_update = WorkflowTaskUpdate(**payload)
     res = await client.patch(
         f"{settings.BASE_URL}/workflow/{id}/edit-task/{workflow_task_id}",
-        json=payload,
+        json=payload_update.dict(exclude_unset=True),
```

Comment:
FYI, on the client side it is not strictly necessary to strip unset attributes. However it does reduce the amount of data that need be moved, so good to go.

Reply:
Seems necessary, if I did not put the `exclude_unset` param it sets `None` all the other params and I receive an error from the backend validation.

Reply:
double thinking about this, you are absolutely right. My bad.